batcher/aligned-batcher/src/types/batch_queue.rs

```diff
@@ -147,6 +147,7 @@ pub(crate) fn try_build_batch(
     batch_queue: BatchQueue,
     gas_price: U256,
     max_batch_size: usize,
+    max_batch_len: usize,
```

Comment:
What would be the difference between `max_batch_size` and `max_batch_len` here?

Reply:
size is of bytes, len if of amount of proofs

Reply:
I would either add a comment or rename to something like `max_batch_proofs`. That would remove the ambiguity. The other parameter could be `max_batch_bytes`.